gapis/api/terminator/terminator.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package terminator
+
+import (
+	"context"
+
+	"github.com/google/gapid/gapis/api"
+	"github.com/google/gapid/gapis/api/transform2"
+)
+
+// Terminator is an Transformer that prevents commands passing-through it after
+// a certain point in the stream.
+type Terminator interface {
+	transform2.Transform
+
+	// Add relaxes the termination limit to pass-through all commands before and
+	// including the command or subcommand.
```

Comment:
I don't understand what this comment is trying to say?

Reply:
This is a move from the previous file. I do not know either. Maybe awolozsyn@ knows.